boards/arm/cy8ckit_062_wifi_bt_m0/doc/cy8ckit_062_wifi_bt_m0.rst

```diff
@@ -0,0 +1,106 @@
+.. _cy8ckit_062_wifi_bt_m0:
+
+PSoC6 WiFi-BT Pioneer Kit
```

Comment:
@dbkinder 
Sorry, but I do not understand your point. Can you provide me more details, please?
It is made similarly to lpcxpresso54114_m0 and lpcxpresso54114_m4 boards.

Reply:
The recommendation is to change the title of these documents to be:   ``PSoC6 WiFi-BT Pioneer Kit (M0)`` and ``PSoC6 WiFi-BT Pioneer Kit (M4)``, as was done for the lpcxpresso54114 boards.  Otherwise, when you look at the table of contents (https://docs.zephyrproject.org/latest/boards/boards.html) the two boards will show the same title.

Reply:
Now I see. Thank you